### PR TITLE
User can search for reviews by business details.

### PR DESCRIPTION
**Pull request by rrundle**

### CODE REVIEW

File: search.js [hunk 1]
```diff
@@ -1,56 +1,220 @@
...
+//disabling searching by zipcode
 var zipSearch = document.getElementById("location");
 zipSearch.setAttribute("disabled", true);
 
+//loading each object in the businesses array on the search page
+//puting it inside a new div called business
 for (var i = 0; i < businesses.length; i++) {
-  var business = document.createElement("div")
+  var business = document.createElement("div");
+  business.setAttribute("id", "business");
```
Comment (thebearingedge) [inline search.js]: HTML `id` attributes should be unique page-wide. For styling, use a CSS class instead. For identification, use a unique value from your data model.
screenshot 2016-11-24 13 35 03

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: search.js [hunk 2]
```diff
@@ -1,56 +1,220 @@
...
+var changeZipDisplay = document.getElementById("results");
+
+//function to clear the list div when search submit button is fired
+function empty(element) {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+}
+
+//declaring function to search for matching text
```
Comment (thebearingedge) [inline search.js:125]: Use meaningful function and parameter names instead of comments.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: search.js (hunk 1 above)
Comment (rrundle) [reply search.js]: Makes sense. Done.

File: search.js (hunk 2 above)
Comment (rrundle) [reply search.js:125]: changed function & variable names but left in comments for clarity.

**Pull request merged**